client/landing/stepper/hooks/use-prepare-site-for-migration.ts

```diff
@@ -44,9 +46,63 @@ export const usePrepareSiteForMigration = ( siteId?: number ) => {
 		enabled: Boolean( pluginInstallationState.completed ),
 	} );
 
+	const [ transferTimingTracked, setTransferTimingTracked ] = useState( false );
```

Comment:
We should use useRef instead of state if the changes do not trigger layout changes.
You can also centralize in a single object state with no issues.

```suggestion
	const [ transferTimingTracked, setTransferTimingTracked ] = useRef( { transferTimingTracked: 0,  siteTransferEnd: 0, ...} );
```

Reply:
I like this way of doing it. But I'll note that I'm going to stick with `Date.now()` since `performance.now()` doesn't work in Firefox, especially if browser-fingerprint protection is enabled.